### PR TITLE
Add environment variable for retrying auto installation

### PR DESCRIPTION
**Pull request by SwikritiT**
This PR https://github.com/juliushaertl/nextcloud-docker-dev/pull/189 made the auto retry of app installation default to 0 so we add the environment variable to `120`.

### CODE REVIEW

File: src/views/ProjectsTab.vue [hunk 1]
```diff
@@ -321,8 +321,8 @@ export default {
 		}
 		&--unlinkactionbutton {
 			position: absolute;
-			right:0;
-			top:0;
+			right: 0;
+			top: 0;
```
Comment (individual-it) [inline src/views/ProjectsTab.vue:325]: this change does not look like it belongs into this PR

**Review by individual-it — CHANGES_REQUESTED** [review]

File: src/views/ProjectsTab.vue (hunk 1 above)
Comment (SwikritiT) [reply src/views/ProjectsTab.vue:325]: I don't know why the stylelint of the CI of previous PR https://github.com/nextcloud/integration_openproject/pull/407 didn't complain about this issue but in this PR https://github.com/nextcloud/integration_openproject/actions/runs/4945800995/jobs/8843015517 it complained so the CI might be red.

Comment (SwikritiT) [reply src/views/ProjectsTab.vue:325]: I've created a seperated PR for this https://github.com/nextcloud/integration_openproject/pull/409. I'll rebase once it's merged

**Review by individual-it — APPROVED** [review]

**Comment by SwikritiT** [comment]
I'm merging this because this https://github.com/nextcloud/integration_openproject/pull/409 won't pass the CI as it needs the variable added by this PR because activity app is not installed without it https://github.com/nextcloud/integration_openproject/actions/runs/4955244949/jobs/8867408836?pr=409#step:18:94

**Pull request merged**